src/main/java/org/jenkinsci/test/acceptance/po/WorkflowJob.java

```diff
@@ -37,4 +47,20 @@ public WorkflowJob(Injector injector, URL url, String name) {
     public final Control script = control("/definition/script");
     public final Control sandbox = control("/definition/sandbox");
 
+    public String copyResourceStep(String filePath) {
+        final Resource res = resource(filePath);
+        try (InputStream in = res.asInputStream()) {
```

Comment:
Is the `resource` method new? I don't see it. Can it return `null` and should there be a `null` check on the `if` statement?

I would add the `null` check anyway (irrespective of `resource`s return being marked `@Nonnull`). I know others wouldn't agree with that but I'd respectfully agree to disagree with them :smile: